### PR TITLE
Add GraphQL ^0.13.0 to the peer dependencies

### PR DESCRIPTION
**Pull request by rodrigues**
```
warning " > graphiql@0.11.11" has incorrect peer dependency "graphql@^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0".
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "markdown-it": "^8.4.0"
   },
   "peerDependencies": {
-    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0",
+    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.1",
```
Comment (IvanGoncharov) [inline package.json]: It should be `^0.13.0`.

Comment (rodrigues) [reply package.json]: @IvanGoncharov done 👍🏼

**Comment by IvanGoncharov** [comment]
@rodrigues Thanks.
One important thing is to update graphql in all dependencies of GraphiQL before merging this PR:
https://github.com/graphql/codemirror-graphql
https://github.com/graphql/graphql-language-service

**Comment by rodrigues** [comment]
@IvanGoncharov these dependencies already have (unmerged) PRs to update graphql:
https://github.com/graphql/codemirror-graphql/pull/244
https://github.com/graphql/graphql-language-service/pull/214

thanks!

**Comment by IvanGoncharov** [comment]
@rodrigues Thanks for the research 👍

@asiandrummer Can you please make a new release?

**Comment by BBonifield** [comment]
Any progress on getting this into a release?

**Comment by ajhyndman** [comment]
@asiandrummer @leebyron This two month latency on a version patch is making my business case for using GraphQL a little harder.  😞

**Comment by jamespolanco** [comment]
It looks like they are getting closer. GraphQL Language Service released a new update (https://github.com/graphql/graphql-language-service/issues/219#issuecomment-396283697) with 0.13 support and now we need code mirror to update https://github.com/graphql/codemirror-graphql/issues/247

**Comment by wmertens** [comment]
It's sad that this is purported to be a community project, but in the end it seems we're at the whims of Facebook's Project Managers deciding who gets to work on what.

I hope you can agree that a trivial PR like this should not take so long to get merged, and it would be great if some outside-FB maintainers would be added, at least to these tangential projects.

Some projects work entirely on community rules, like [0mq](http://zeromq.org/docs:contributing) and it seems to work out well.

**Comment by mattbaker** [comment]
@asiandrummer @leebyron — bump

**Comment by AGS-** [comment]
Hey everyone, sorry this PR has become a little stale. Let me look into what it takes to get this merged. @rodrigues, thanks for taking the time to make this contribution.

**Comment by AGS-** [comment]
Merged!

**Comment by mwld** [comment]
Thanks for merging! When will the next release be done @AGS-?

**Comment by AGS-** [comment]
Uh, I have to cut a release of codemirror-graphql and graphql-language-service before and also go through the remaining PRs, so let's aim for sometime late next week?

**Comment by irace** [comment]
Great - thank you!

**Pull request merged**